requirements/app.txt

```diff
@@ -2,8 +2,8 @@ galaxy.plugin.api==0.69
 
 # steam_network backend
 protobuf==3.19.4
```

Comment:
With better proto, we can update this to a much newer version as well, though i'm still working out the kinks there.

Reply:
We could use a newer protobuf, too, if Gog wouldn't still use python 3.7 for which Protobuf 3.19 is the latest compatible version

Reply:
Yeah, it's not ideal. As i mentioned before I'd prefer python 3.10+ because i want `match` aka a switch statement (but python is too much of a hipster to call it that).